$NetBSD: patch-pcre_configure.in,v 1.2 2020/04/20 14:47:35 micha Exp $

Replace nonportable operator "==" with POSIX equivalent.
Reported and fixed upstream (remove this patch for next release).

--- pcre/configure.in.orig	2017-05-03 17:47:33.000000000 +0000
+++ pcre/configure.in
@@ -183,7 +183,7 @@ dnl Handle --enable-ebcdic
 
 AC_ARG_ENABLE(ebcdic,
 [  --enable-ebcdic         assume EBCDIC coding rather than ASCII],
-if test "$enableval" == "yes"; then
+if test "$enableval" = "yes"; then
   EBCDIC=-DEBCDIC=1
 fi
 )
